src/SFA.DAS.ApplyService.InternalApi/Controllers/RoatpModeratorController.cs

```diff
@@ -0,0 +1,143 @@
+        {
+            var sectors = await _sectorService.GetSectorsForModerator(applicationId, request.UserId);
+
+            return sectors.OrderBy(sec => sec.PageId).ToList();
+        }
+
+        [HttpGet("Moderator/Applications/{applicationId}/SectorDetails/{pageId}")]
+        public async Task<AssessorSectorDetails> GetSectorDetails(Guid applicationId, string pageId)
+        {
+            return await _sectorDetailsService.GetSectorDetails(applicationId, pageId);
+        }
+
+        [HttpGet("Moderator/Applications/{applicationId}/Sequences/{sequenceNumber}/Sections/{sectionNumber}/Page")]
+        public async Task<AssessorPage> GetFirstModeratorPage(Guid applicationId, int sequenceNumber, int sectionNumber)
+        {
+            return await GetModeratorPage(applicationId, sequenceNumber, sectionNumber, null);
+        }
+
+        [HttpGet("Moderator/Applications/{applicationId}/Sequences/{sequenceNumber}/Sections/{sectionNumber}/Page/{pageId}")]
+        public async Task<AssessorPage> GetModeratorPage(Guid applicationId, int sequenceNumber, int sectionNumber, string pageId)
+        {
+            return await _pageService.GetPage(applicationId, sequenceNumber, sectionNumber, pageId);
+        }
+
+        [HttpGet("Moderator/Applications/{applicationId}/Sequences/{sequenceNumber}/Sections/{sectionNumber}/Page/{pageId}/BlindAssessmentOutcome")]
+        public async Task<BlindAssessmentOutcome> GetBlindAssessmentOutcome(Guid applicationId, int sequenceNumber, int sectionNumber, string pageId)
+        {
+            var blindAssessmentOutcome = await _mediator.Send(new GetBlindAssessmentOutcomeRequest(applicationId, sequenceNumber, sectionNumber, pageId));
+
+            return blindAssessmentOutcome;
+        }
+
+        [HttpPost("Moderator/Applications/{applicationId}/SubmitPageReviewOutcome")]
+        public async Task SubmitPageReviewOutcome(Guid applicationId, [FromBody] SubmitPageReviewOutcomeCommand request)
+        {
+            await _mediator.Send(new SubmitModeratorPageOutcomeRequest(applicationId, request.SequenceNumber, request.SectionNumber, request.PageId, request.UserId, request.Status, request.Comment, request.ExternalComment));
+        }
+
+        [HttpPost("Moderator/Applications/{applicationId}/GetPageReviewOutcome")]
+        public async Task<ModeratorPageReviewOutcome> GetPageReviewOutcome(Guid applicationId, [FromBody] GetPageReviewOutcomeRequest request)
+        {
+            var pageReviewOutcome = await _mediator.Send(new GetModeratorPageReviewOutcomeRequest(applicationId, request.SequenceNumber, request.SectionNumber, request.PageId, request.UserId));
+
+            return pageReviewOutcome;
+        }
+
+        [HttpPost("Moderator/Applications/{applicationId}/GetPageReviewOutcomesForSection")]
+        public async Task<List<ModeratorPageReviewOutcome>> GetPageReviewOutcomesForSection(Guid applicationId, [FromBody] GetPageReviewOutcomesForSectionRequest request)
```

Comment:
same question as before about using a POST

Reply:
Copied across from Assessor which did it in the same way